usr/share/lib/img_proof/tests/SLES/test_sles_hardened.py

```diff
@@ -0,0 +1,69 @@
+import pytest
+
+
+SWAP_FILE = "/swap"
+
+
+# Workaround for https://github.com/OpenSCAP/openscap/issues/1796
+def setup_swap(host, swap_file=SWAP_FILE):
+    min_memory = 16e6
+    mem_total = int(host.run(
+        "cat /proc/meminfo | awk '$0 ~ /MemTotal:/ { print $2 }'"
+    ).stdout.strip())
+    if mem_total < min_memory:
+        # Follow steps in https://btrfs.readthedocs.io/en/latest/Swapfile.html
+        for command in [
+            "sudo truncate -s 0 {}".format(swap_file),
+            "sudo chattr +C {} || true".format(swap_file),
+            "sudo fallocate -l 4G {}".format(swap_file),
+            "sudo chmod 600 {}".format(swap_file),
+            "sudo mkswap {}".format(swap_file),
+            "sudo swapon {}".format(swap_file),
+        ]:
+            result = host.run(command)
+            if result.rc != 0:
+                return False
+    return True
+
+
+def bye_swap(host, swap_file=SWAP_FILE):
+    host.run("sudo swapoff {}".format(swap_file))
+    host.run("sudo rm {}".format(swap_file))
+
+
+def test_sles_hardened(host, get_release_value, is_sles_sap):
+    variant = get_release_value('VARIANT_ID') or ''
+
+    if "hardened" not in variant:
+        pytest.skip('Skipping non-hardened image')
+
+    if not setup_swap(host):
+        pytest.skip("Failed to setup swap, not enough memory.")
+
+    xml_path = "pub/projects/security/oval/suse.linux.enterprise.15.xml"
```

Comment:
This makes the assumption that all our images have `suse.linux.enterprise.15.xml` as a common profile. While this is true today for SLES and SLES4SAP. This will not be true when we publish a hardened SLE_Micro image. We should have a way to associate a profile with the distribution where the test is running.

Reply:
This file is in the SLES directory, so it's a safe assumption for now.  When will the image be published anyway? Can we fix this later when it happens?